frontend/src/css/modules/base.scss

```diff
@@ -415,6 +415,7 @@ nav {
 .btn {
     text-transform: none;
     font-family: 'Roboto';
+    border-radius: 20px;
```

Comment:
@S-ulphuric There is already a class '.grad-btn' please add that and remove this style form here. You have to find those buttons to make these required changes.

Reply:
Ok. Thanks